library/Backends/llvm/src/Juvix/Backends/LLVM/Parameterization.hs

```diff
@@ -52,7 +53,12 @@ llvm =
     builtinTypes :: Param.Builtins PrimTy
     builtinTypes =
       [ ("LLVM.int8", PrimTy LLVM.i8),
-        ("LLVM.int16", PrimTy LLVM.i16)
+        ("LLVM.int16", PrimTy LLVM.i16),
+        ("LLVM.string", PrimTy (Codegen.Type.pointerOf LLVM.i8)),
+        -- PLEASE FIX --
+        ("LLVM.set", PrimTy (Codegen.Type.pointerOf LLVM.i8)),
+        -- PLEASE FIX --
+        ("LLVM.list", PrimTy (Codegen.Type.pointerOf LLVM.i8))
```

Comment:
Why is this added in this PR?
Why are the pointers fixed to be of type i8? How would this work on other targets / architectures? Is this the thing to fix?

Reply:
> Why is this added in this PR?
The codebase refuses to build without adding these in the latest pr, thus it must be addressed

> Why are the pointers fixed to be of type i8? How would this work on other targets / architectures? Is this the thing to fix?
I'ts fine to represent strings as pointers to `i8`, as when we consider ascii chars they are i8, ideally we should probably use `i16` to fit alll of unicode, however that is not here nor there. As for architectures, it should not matter, as LLVM has integers as primitives and can compile this to all sorts of architectures.

The `PLEASE FIX` is for sets and lists at we should not encode them this way in any functional way, unlike strings which can work with ascii